.github/workflows/build-and-push.yml

```diff
@@ -3,6 +3,8 @@ name: Build and Push
 on:
   push:
     branches: [ main ]
+    tags:
+      - v*
```

Comment:
Everything looks good. Just wonder if we should use the release trigger (https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#release) instead of tags? It feels a bit more formal to tie the workflow with release events.

Reply:
It's okay now we follow how kserve is doing, using tags. We might want to consider the release events in the future.